### ISSUE TITLE
Slightly adjust "move book" help text in the version without the grab button

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#137]
A more explicit (i.e., clearer) help text about being on the book to rub might help users understand thsi custom interaction a little better. They have to be on the book to before beginning to rub, so telling them exactly that might be just what they need.

**Current "move book" help text:**
- Use arrow keys, or letter keys W, A, S, or D to move book or zoomed-in book up, left, down, or right.

**Make it more explicit by changing to:** 
- Once on book or zoomed-in book, use arrow keys, or letter keys W, A, S, or D to move up, left, down, or right.

I've added this phrase to the design document, too, for safe keeping.

@zepumph, this should be a tiny issue. Not sure I need to verify. Once implemented, I think we might be ready to port to sonified version.

**Comment by zepumph** [comment]
Implemented above, haven't yet moved it to the sonified version
